src/AddTodoForm.jsx

```diff
@@ -14,21 +16,25 @@ const AddTodoForm = ({ onAddTodo }) => {
         event.preventDefault();
         onAddTodo(todoTitle);
         setTodoTitle(''); //resets 
+        setRating(0);
     };
 
+
     return (
-        <form onSubmit={handleAddTodo}>
+        <form onSubmit={handleAddTodo} className='todoForm"' >
 
             <>
                 <InputWithLabel
                     todoTitle={todoTitle}
+                    className='title'
                     handleTitleChange={handleTitleChange}
                 >
-                    Title
+                    Book Title
                 </InputWithLabel>
+                <RatingReview rating={rating} setRating={setRating} /> {/* Rating input */}
```

Comment:
Nice experimenting here with other UI elements! Make sure you take some time to look at the react-rating-stars code to ensure that you understand what you're including in your codebase here. Do you think you could build your own rating component? I'm sure you could ;)